docs/api/core-node/get-account-data.example.json

```diff
@@ -0,0 +1,6 @@
9e99cf577accc33a7ddfca9667f19a78184fa0601fcc19756b78b5700e2d1991e11e08ad98d7ece28de96517515b03584861ab80d21ae5fe0cac048cd9c45f5c06eb75ea2caaab7129285c259e5f97ab81ebd01fc6260e0405151be54e1fba360e8a24ac2abcd35e525ea7eb547da1918248c98295ea279b689e9fcd99508494accd87dbf20acb955ab76666f2a38a2451aecdc4ec9cb74cfe56d1ccfcbdfda8d443f423f095ae52fae7c283201b634bcd3cc576801829756b92e82e34f92e6c66ee984d751a1c5e9341c23e823f0f84c71c07c4cf46b1de283f413830cd082753e199758b4f6a8e6160577e94332c6ffb9cef32f4b5002e0baf5163a9d56cc7183fceb382e5d25a9bd7c87daeb01f17c7de4a77756394282c5b2c3cedf4e4fb639d1fd1d584fa88d9760f3e3d4886ca030d10a00e909455dc1b960a1342613f265700c49d42bf22aa264ac6fb9e82d661dadfd65db3a81313fb86e039909f11c0db1e4a4781f5930418a8ba7bed12b2bc76fa96f2bf7150f9b1fab7e5fa4a9958210a78cf2ff775e1420476d08b23bfba6426c7b0d1cef1381c9f200223bc18f790d225617a0860079d6d117d70ee2d6ffb128103d2236a4faaa812007ab9979873afc207eb8fff78c677420be64a7ef352cfae1fa3611f2b859ad1134bbfeb460d816f698689c337da43ad88578a7068254a22de6dcec9fa8092916913c8f8fbd3dafc67ceaea2c8621837ede3a19e4fe8e9e2b5265ba72b2b5e99cb16f92723cc7ea0cfaff7ace8b68f9e3f7fe4a876a0c6275bb98939599c00d2eb53cd7f82defa6d8ed5637ac1e2eec9c0b61a626262f03d2a9f605ff0ff07ee88ad4bf9a5460234804f537edd856d4b1a76b1eafccf4c01637972bad004e808599bb1cf5380742f4452fc59582219659fd88a7068254a22de6dcec9fa8092916913c8f8fbd3dafc67ceaea2c8621837ede37f6b28ed8ce86cb36e85b909f6d7a57c4bda894fc6f118410d61e1ee3a59e03ddefa4bccb1ebd68f7b125ce6486c583df2fc010e9d44f0e7a4f3bbb8753b45e5855e5776cafce4422653bb490d2170413402c3a296216c34bdabf8677c80af927764770746827ad77a64964d1cde5dfc973f78ae6f08161f3f28ba7aa077e563308e9dae6c469bec6822ead3b51dd1234250bc44fd83a5ab73e29c982ca46668303e8003c3fb176af5d51330fc452824593c2b1b0f1afa4d4174fda58f639b85a2d0a7007a37ffd454b7b35397e20dd1a60866b3af2c365404bd03024301a60b70db8b730e37d967078b922c71a7e0fcae3539665aeff5adc9a101a0110337a5909f5e8d56428e77bd8131ebfa68f2f3bf90bed70e7ea5f5cb23553ac1a517d6ceba5ad141b3a1275322d46475e422b0ec6420270b39d67f987f5c8f7813abbd2203a08f22296421801d473d804b19c4bbcd6132e9100f2ec9aa70b1cd1071b37ce117c0f1191a97b44df912e762f1748ab543e0b40d072293a5a03d51f3bf365e9ce6ffd599422826fcdbd2003871b353002fad638d0f42d36d9db7b39b33ea8e65e4aa01a76734464b908db04e73038040df3d703b65357d0fdbe5bc83907c0500000000000000030000000b2703ee01f3db0cbd3e1ba08405564eed2671533e2509dfc19cc1e1c18fa1fde03868eb120025526d5a06f93353ae65afd8a923c3704fbc603fde398981ab5a55b96f80eaf1221ca0998971764a58269bce1081a3513412213db789632400a9665f2749c93ed0e45184f16b4427948ff4f5c3d50da1aa5567ad850ff18509322676e12142567022b40231ec4e1be7ff397eb842e98a88bc2ece30b25e17c60a372d3e0dc86a3c1c61022db95424b40f156c2f8c5fe0641e981f7c4994c861b22c4ab8fb1adfc2ecca576c7f3a5e43c53e84ea315da76fca30b3e828ff18ee4a491d10d4aecb60a2ddb360e473010e37a6ad59962661b774ea793fb08fb0115cbdef31f3b08bb4d1c350a8b32ff847121f1f4eab06c5298e3047dc648f7d62436a5014562ea1b51d82a31aa6d639f09ae449b4b7a7d55b389bb97b053539b1d3e68ae8ce791ded714350333cd5092df076208dd9a40b965afb01d5c486ff2fb044",
```

Comment:
Should we make these long arbitrary hex strings into something like `"balance_proof": "0xabcd"`, for file readability?

Reply:
We should, I'll make the change :)